jwst/photom/photom.py

```diff
@@ -634,7 +649,24 @@ def photom_io(self, tabdata, order=None):
 
         """
         # Get the scalar conversion factor from the PHOTMJSR column of the table row
-        conversion = tabdata['photmjsr']
+        try:
```

Comment:
Maybe in the docs include a short explanation of what's going on in this new code.

Reply:
Good point, thanks!  Yes, I'll update the docs.